New log file opened at Mon Jan 16 19:13:08 CET 2017 by 8388@Besitzer-PC
Taking next connection to: 57050
New client attempting to connect on port: 57051-57050
  Address = /192.168.0.102:57050
This client on port 57051 has uuid e3c67592-a268-4a6d-a317-491f26344ec5 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-e3c67592-a268-4a6d-a317-491f26344ec5
Taking next connection to: 57050
New client attempting to connect on port: 57052-57050
  Address = /192.168.0.102:57050
Handshake exception on socket: 57052
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57050
New client attempting to connect on port: 57053-57050
  Address = /192.168.0.102:57050
This client on port 57053 has uuid 13e3d898-6d40-4a7c-8179-d813b46bd929 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-e3c67592-a268-4a6d-a317-491f26344ec5, activator-13e3d898-6d40-4a7c-8179-d813b46bd929
Taking next connection to: 57050
New client attempting to connect on port: 57059-57050
  Address = /192.168.0.102:57050
Handshake exception on socket: 57059
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57050
New client attempting to connect on port: 57060-57050
  Address = /192.168.0.102:57050
This client on port 57060 has uuid e8b376d6-584b-4943-9151-8418ab696f54 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-e3c67592-a268-4a6d-a317-491f26344ec5, play-fork-run-e8b376d6-584b-4943-9151-8418ab696f54
Taking next connection to: 57050
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57050
